acceptance/tests/resource/exec/should_run_command_in_cwd.rb

```diff
@@ -120,7 +120,10 @@ def assert_file_on(host, filepath, failure_comment)
         # we need to create the user directory ourselves in order for deb users to successfully login
         on(agent, "mkdir /home/#{username} && chown -R #{username} /home/#{username}")
       end
-      teardown { agent.user_absent(username) }
+      teardown do
+        on(agent, "pkill -9 -u `id -u #{username}`") if agent.platform =~ /ubuntu-22\.04/
```

Comment:
I think `pkill -9 -u #{username}` will accept the symbolic name so you don't have to resolve it yourself... But I don't think the `pkill` command exists on all platforms we test against (thinking about AIX, Solaris, Windows, maybe macOS)?

Reply:
So maybe only execute this if `which pkill` returns 0?